src/nashpy/game.py

```diff
@@ -423,3 +424,62 @@ def linear_program(self):
         row_strategy = linear_program(row_player_payoff_matrix=A)
         column_strategy = linear_program(row_player_payoff_matrix=B.T)
         return row_strategy, column_strategy
+
+    def regret_minimization(self, learning_rate=0.1, max_iterations=100):
+        """
+        Build best Strategies probability of both players using regret minimization method
+        Algorithm implemented here is Algorithm 4.3 Theorem 4.4 of [Nisan2007]_
+
+        Parameters
+        ----------
+
+        learning_rate: float ( Optional Defaulted to 0.1 )
+            The  learning_rate determines the magnitude of the update towards the regrets
+            The learning rate scales the regrets before they are added to the current strategy.
+            A higher learning rate results in a larger update, while a lower learning rate leads to a smaller update.
+            This value allows you to control the pace towards a Nash equilibrium.
+
+        max_itreations: Integer ( Optional Defaulted to 100 )
+            This value is defaulted to 100 itrations, this number could be modified to a larger or smaller number based on the untilities/payoff matrix shape
+        Returns
+        -------
+        tuple
+            The Nash equilibria
+        """
+        A, B = self.payoff_matrices
+        return regret_minimization(
+            A=A, B=B, learning_rate=learning_rate, max_iterations=max_iterations
+        )
+
+    def imitation_dynamics(
+        self,
+        population_size=100,
+        num_of_generations=1000,
```

Comment:
```suggestion
        iterations=1000,
```

Reply:
Also changed, in the test, functions, docs, how-to and in the game code.